src/components/SlidePanel/_slide-panel.scss

```diff
@@ -0,0 +1,21 @@
+@import "vanilla-framework/scss/vanilla";
+
+.slide-panel {
+  $top-header-height: 40px;
+
+  background-color: $color-x-light;
+  box-shadow: $box-shadow;
+  height: calc(100% - #{$top-header-height});
+  overflow: scroll;
+  position: fixed;
+  right: 0;
+  top: $top-header-height;
```

Comment:
Can you add margin-left that matches the width of the collapsed navigation panel so this UI works nicely on small screen.

Reply:
done